### PR TITLE
Refactor all instances of Person to Food

### PR DESCRIPTION
**Pull request by seanlowjk**
Refactor all instances of Person to Food in fxml, java and txt files.

### CODE REVIEW

File: docs/LearningOutcomes.adoc [hunk 1]
```diff
@@ -0,0 +1,216 @@
+= Learning Outcomes
```
Comment (robytanama) [inline docs/LearningOutcomes.adoc]: This part can delete?

Comment (seanlowjk) [reply docs/LearningOutcomes.adoc]: Yes I have deleted it already, thank you for the reminder!

File: docs/diagrams/tracing/LogicSequenceDiagram.puml [hunk 2]
```diff
@@ -13,7 +13,7 @@ create ecp
 abp -> ecp
 abp -> ecp ++: parse(arguments)
 create ec
-ecp -> ec ++: index, editPersonDescriptor
+ecp -> ec ++: index, EditFoodDescriptor
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply docs/diagrams/tracing/LogicSequenceDiagram.puml]: Dude that's the UML diagram LOL

File: docs/templates/LICENSE [hunk 3]
```diff
@@ -5,11 +5,11 @@ MIT License
 
 Copyright (C) 2012-2018 Dan Allen, Ryan Waldron and the Asciidoctor Project
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any food obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit foods to whom the Software is
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply docs/templates/LICENSE]: Will edit , thanks for the comment!

File: docs/diagrams/tracing/LogicSequenceDiagram.puml (hunk 2 above)
[the remark this answers is not in the document]
Comment (robytanama) [reply docs/diagrams/tracing/LogicSequenceDiagram.puml]: OOPS!!

File: docs/templates/LICENSE (hunk 3 above)
[the remark this answers is not in the document]
Comment (robytanama) [reply docs/templates/LICENSE]: Thank you for explaining yourself

**Pull request merged**